lib/fog/vsphere/requests/compute/list_vm_volumes.rb

```diff
@@ -28,6 +28,8 @@ class Real
         #  unitNumber: 1
         # )]
 
+        # rubocop:disable Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/MethodLength
```

Comment:
@sbernhard could you pleaser remove these? It's not necessary because of config in `.rubocop.yml`, one is disabled and one is set on very high number, so we don't need to disable it localy. That should enable the tests to at least run :)

Reply:
The codeclimate will just complain anyway, but we have so many issues in this repo, that its not real problem.